docs/about/whats-new.ipynb

```diff
@@ -0,0 +1,958 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = sc.DataArray(data=sc.zeros(dims=['x','day'], shape=(4,3)))\n",
+    "da.coords['x'] = sc.linspace(dim='x', unit='m', start=0.1, stop=0.2, num=5)\n",
+    "da.coords['day'] = sc.array(dims=['day'], values=[1, 7, 31])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da['day', sc.scalar(7)]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da['x', 0.13*sc.units.m]  # selects bin containing this value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Support for datetime64\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.6**\n",
+    "    \n",
+    "- Previously we stored time-related information such as, e.g., sample-temperature logs as integers.\n",
+    "- Added support for datetime64 compatible with [np.datetime64](https://numpy.org/doc/stable/reference/arrays.datetime.html)\n",
+    "- Time differences (`np.timedelta64`) are not used, we simply use integers since in combination with scipp's units this provides everything we need.\n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "Example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.array(dims=['time'], values=np.arange(\n",
+    "    np.datetime64('2021-01-01T12:00:00'),\n",
+    "    np.datetime64('2021-01-01T12:04:00')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Datetimes and intgers with time units interoperate naturally.\n",
+    "We can offset a datetime by adding a duration:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var + 123 * sc.Unit('s')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or subtract datetimes to obtain a duration:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var['time', 10] - var['time', 0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`to_unit` can be used to convert to a different precision:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.to_unit(var, 'ms')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Operations\n",
+    "\n",
+    "#### Creation functions\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.5**\n",
+    "\n",
+    "For convenience and similarity to `numpy` we added [functions that create variables](https://scipp.github.io/python-reference/api.html#creation-functions).\n",
```

Comment:
Links should refer to the rst sources so that Sphinx can verify them. This link is broken and there might be others.